src/helpers/migrations.js

```diff
@@ -1,6 +1,5 @@
-/* eslint-disable import/prefer-default-export */
-import log from 'electron-log';
 import { isEdge, isNode } from 'react-flow-renderer';
```

Comment:
Removing the eslint-diable here will cause an error.

Reply:
Whoops, I thought I undid that